inquirer/render/console/_checkbox.py

```diff
@@ -14,35 +14,39 @@ def __init__(self, *args, **kwargs):
     @property
     def is_long(self):
         choices = self.question.choices or []
-        return len(choices) >= MAX_OPTIONS_DISPLAYED_AT_ONCE
+        return len(choices) >= self.MAX_OPTIONS_DISPLAYED_AT_ONCE
 
     def get_options(self):
         choices = self.question.choices or []
         if self.is_long:
             cmin = 0
-            cmax = MAX_OPTIONS_DISPLAYED_AT_ONCE
+            cmax = self.MAX_OPTIONS_DISPLAYED_AT_ONCE
 
-            if half_options < self.current < len(choices) - half_options:
-                cmin += self.current - half_options
-                cmax += self.current - half_options
-            elif self.current >= len(choices) - half_options:
-                cmin += len(choices) - MAX_OPTIONS_DISPLAYED_AT_ONCE
+            if self.half_options < self.current < len(choices) - self.half_options:
+                cmin += self.current - self.half_options
+                cmax += self.current - self.half_options
+            elif self.current >= len(choices) - self.half_options:
+                cmin += len(choices) - self.MAX_OPTIONS_DISPLAYED_AT_ONCE
                 cmax += len(choices)
 
             cchoices = choices[cmin:cmax]
         else:
             cchoices = choices
 
-        ending_milestone = max(len(choices) - half_options, half_options + 1)
-        is_in_beginning = self.current <= half_options
-        is_in_middle = half_options < self.current < ending_milestone
+        # Check if length is even or odd
+        if self.MAX_OPTIONS_DISPLAYED_AT_ONCE % 2 == 0:
+            ending_milestone = max(len(choices) - self.half_options, self.half_options + 1)
+        else:
+            ending_milestone = max(len(choices) - self.half_options, self.half_options)
```

Comment:
Since you introduces a new code branch here, would be nice to have a test case when self.MAX_OPTIONS_DISPLAYED_AT_ONCE is odd.

Reply:
@domwhewell  this IF statement is new on the code, so we should have one test that goes to the IF and on that goes to ELSE to maintain the code coverage of test.